src/flatten.js

```diff
@@ -0,0 +1,18 @@
+export default function flatten(arr) {
```

Comment:
Is this really so deep that the standard recursive approach doesn't work?

```js
function flatten(arr, result = []) {
  for (const el of arr) {
    if (!Array.isArray(el)) {
      result.push(el);
    } else {
      flatten(el, result);
    }
  }

  return result;
}
```

Reply:
Idk it's forked from enzyme and react

Reply:
ahhh okay

Reply:
I should probably make that clear tho